src/notifications/notificationsView.ts

```diff
@@ -0,0 +1,111 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { dispose } from '../common/utils';
+import { NotificationSubjectType } from '../github/interface';
+import { IssueModel } from '../github/issueModel';
+import { PullRequestModel } from '../github/pullRequestModel';
+import { NotificationsProvider } from './notificationsProvider';
+import { LoadMoreNotificationsTreeItem, NotificationsSortMethod, NotificationTreeDataItem, NotificationTreeItem } from './notificationTreeItem';
+
+const devMode = false; // Boolean("true");
```

Comment:
A comment like [this](https://github.com/microsoft/vscode/blob/f79f2dca07abe34a0c6314f63cef13753b8aa4a0/src/vs/base/common/observableInternal/index.ts#L22-L25) would be useful